src/components/FieldLabel/FieldLabel.tsx

```diff
@@ -3,9 +3,14 @@ import cx from "classnames";
 import Icon from "../Icon/Icon";
 import VibeComponentProps from "../../types/VibeComponentProps";
 import "./FieldLabel.scss";
+import { descriptionTypesMap } from "src/storybook/components/related-components/component-description-map";
```

Comment:
I didn't understand why we have here this import?

Reply:
Agree, it's looks like a mistake. let's remove it?